stingray/powerspectrum.py

```diff
@@ -269,7 +253,7 @@ def compute_rms(
             )
 
     def _rms_error(self, powers):
-        """
+        r"""
```

Comment:
No, it avoids the \s and \o strings to be substituted with special characters by the Python interpreter, so that they are correctly passed to the LaTeX interpreter when the documentation is compiled